man/citgm-all.1

```diff
@@ -70,6 +70,12 @@ Run tests in parallel (automatically detect core count)
 .TP
 .BR \-\-tmpDir " " \fI<path>\fR
 Directory to test modules in
+.TP
+.BR \-\-withTags " " \fI<tag1,tag2>\fR
+Only test modules from the lookup that contain a matching tag field
+.TP
+.BR \-\-excludeTags " " \fI<tag1,tag2>\fR
+Specify which tags to skip from the lookup
```

Comment:
These need to be updated if you change the equivalents in the README (a linter or even a test for this would be great).